hugolib/skeletons/theme/layouts/partials/head/js.html

```diff
@@ -0,0 +1,3 @@
+{{ with resources.Get "js/main.js" | js.Build | minify | resources.Fingerprint }}
```

Comment:
We can certainly take several steps to get this right, but I think this is important enough to adjust. Fingerprinting during development is (usually) a bad idea.

I'm not sure what would be the general good construct here, but I suspect this construct should be OK:

```
{{ $js :=  resources.Get "js/main.js" | js.Build  }}
{{ if ne hugo.Environment "development" }}
{{ $js = $js | minify | fingerprint }}
{{ end }}
```

The same applies to similar constructs above.

On a more nitpicking point: I much prefer using the built-in minifier in ESBuild.